main.cpp

```diff
@@ -0,0 +1,130 @@
+#include <iostream>
+#include <string>
+#include <stack>
+
+/**
+ * @brief Get and pop last two items on the stack. This is done because std::stack::pop does not return value that is removed
+ *
+ * @param stack
+ * @param a
+ * @param b
+ */
+void getTwoTopItems(std::stack<int> &stack, int &a, int &b)
+{
+    a = stack.top();
+    stack.pop();
+    b = stack.top();
+    stack.pop();
+}
+
+int main()
+{
+    //stack with the actual values
+    std::stack<int> stack = std::stack<int>();
+    //simple flag to allow exiting from the app
+    bool working = true;
+    //input string that will be written into by user each iteration
+    std::string input;
+    std::cout << "add, sub, mul, div, exit, peak, swap, pop, or the number to add: " << std::endl;
+    while (working)
+    {
+        std::cin >> input;
+        int a, b;
+        if (input == "add")
+        {
+            if (stack.size() < 2)
+            {
+                std::cout << "Not enough items on stack";
+            }
+            else
+            {
+                getTwoTopItems(stack, a, b);
+                stack.push(a + b);
+            }
+        }
+        else if (input == "sub")
+        {
+            if (stack.size() < 2)
+            {
+                std::cout << "Not enough items on stack";
+            }
+            else
+            {
+                getTwoTopItems(stack, a, b);
+                stack.push(a - b);
+            }
+        }
+        else if (input == "mul")
+        {
+            if (stack.size() < 2)
+            {
+                std::cout << "Not enough items on stack";
+            }
+            else
+            {
+                getTwoTopItems(stack, a, b);
+                stack.push(a * b);
+            }
+        }
+        else if (input == "div")
+        {
+            if (stack.size() < 2)
+            {
+                std::cout << "Not enough items on stack";
+            }
+            else
+            {
+                getTwoTopItems(stack, a, b);
+                stack.push(a / b);
+            }
+        }
+        else if (input == "exit")
+        {
+            working = false;//might as well use break; tbh
```

Comment:
please remove comment, break should not be used

Reply:
In this case it should 
![image](https://user-images.githubusercontent.com/36876492/180379464-3e6b2ea0-585d-4908-8e4c-6eedd60de0c3.png)

(simply because here this is exactly what the `working` flag is  used for, to *break* out of loop)

Reply:
break should not be used in a loop, makes it unclear, while in this case it helps make it clearer, but if its gonna be like that, its only minor

Reply:
debatable but sure